feat(docs): update SDK pages to reflect new CLI (#18903)
expo/expo: docs/pages/build-reference/ios-capabilities.md

To see all of the currently enabled capabilities, visit [Apple Developer Console][apple-dev-console], and find the bundle identifier matching your app, if you click on it you should see a list of all the currently enabled capabilities.

## Manual setup

There are two ways to manually enable Apple capabilities, both systems will require any existing Apple provisioning profiles to be regenerated.

### Xcode

> Preferred method for bare workflow or projects that do **not** using [Expo Prebuild](/workflow/prebuild) to continuously generate the native `ios` and `android` directories.

1. Open the `ios/` directory in Xcode with `xed ios`. If you don't have an `ios/` directory, run `npx expo prebuild -p ios` to generate one.
2. Then follow [this guide][apple-enable-capability].

### Apple Developer Console

First step is to add the respective key/value pairs to your `ios/[app]/[app].entitlements` (or more specific entitlements file for multi-target apps). You can refer to [Supported Capabilities](#supported-capabilities) to determine which entitlements keys should be added.

> 1. Log into [Apple Developer Console][apple-dev-console]. Go to "Certificates, Identifiers, & Profiles" > "Identifiers".

Reviewer: ```suggestion
1. Log into [Apple Developer Console][apple-dev-console]. Click on "Certificates, IDs & Profiles", then navigate to "Identifiers" page.
```

Wording nit. Also, can't we also use link in there? 
* https://developer.apple.com/account/resources/identifiers/list

Author: this is the only link I use for apple dev console, it's on `apple-dev-console`

2. Choose the bundle identifier that matches your app's bundle identifier.
3. Scroll down and enable a capability, some capabilities may require extra setup.
4. Scroll to the top and press "Save". You will see a dialog that says "Modify App Capabilities", press "Confirm" to continue. You will need to regenerate any provisioning profiles that use this bundle identifier before they'll be valid for building a code signed production ipa.

> If the last step is not done correctly then your iOS native build will fail with an error that looks something like:

Reviewer: ```suggestion
If adding capabilities process has not been done correctly then your iOS native build will fail with an error that looks something like:
```

Generalization & wording.
